lib/web/apiserver_test.go

```diff
@@ -3965,6 +3967,137 @@ func TestChangeUserAuthentication_WithPrivacyPolicyEnabledError(t *testing.T) {
 	require.True(t, apiRes.PrivateKeyPolicyEnabled)
 }
 
+func TestChangeUserAuthentication_settingDefaultClusterAuthPreference(t *testing.T) {
+
+	tt := []struct {
+		cloud               bool
+		name                string
+		numberOfUsers       int
+		resultConnectorName string
+	}{{
+		cloud:               true,
+		name:                "cloud, 1 user; signing with passwordless should change connectorName to `passwordless`",
+		numberOfUsers:       1,
+		resultConnectorName: constants.PasswordlessConnector,
+	}, {
+		cloud:               false,
+		name:                "not cloud, 1 user; signing with passwordless should not change connectorName",
+		numberOfUsers:       1,
+		resultConnectorName: "",
+	}, {
+		cloud:               true,
+		name:                "cloud, 2 users; signing with passwordless should not change connectorName",
+		numberOfUsers:       2,
+		resultConnectorName: "",
+	}}
```

Comment:
Please add a tests for different pre-existing connector names, as mentioned below.

Reply:
Yes. We want to not change config that aren't "default", so I think it's worth capturing that in the tests.